### ISSUE TITLE
Service Validator does not verify Excerpt objects against non-excerpt object

### ISSUE THREAD

**Issue by edtanous** [DMTF/Redfish-Service-Validator#602]
Schema items tagged with Redfish.Excerpt objects are intended to contain a schema-defined subset of properties from the Base schema, and include them in the base object.

Redfish service validator should verify that the properties in Excerpt are a subset of the properties present in the source pointed to by DataSrcUri.

**Comment by mraineri** [comment]
I agree; currently in the spec we don't have this semantic called out, so I'd like to discuss this internally to see if we can add to the spec accordingly. But that really is the intent for excerpts.

This may require more work than other changes since this is now cross referencing two resources, so it may take some time to support. But it would be good to keep this visible.
